### PR TITLE
Add Costa（光谷世界城）

### PR DESCRIPTION
**Pull request by lvwzhen**

### CODE REVIEW

File: wuhan.geojson [hunk 1]
```diff
@@ -1,45 +1,80 @@
...
-        "marker-symbol": "cafe"
-      },
-      "geometry": {
-        "type": "Point",
-        "coordinates": [
-          114.392807,
-          30.509323
-        ]
-      }
+  "features": [{
```
Comment (xhacker) [inline wuhan.geojson]: Can you keep the formatting for existing coffee shops?

File: wuhan.geojson [hunk 2]
```diff
@@ -1,45 +1,80 @@
...
-      }
+    "geometry": {
+      "type": "Point",
+      "coordinates": [
+        114.392807,
+        30.509323
+      ]
     }
-  ]
+  }, {
```
Comment (xhacker) [inline wuhan.geojson]: `{` should be in the next line...

File: wuhan.geojson [hunk 3]
```diff
@@ -1,45 +1,80 @@
...
+      "marker-symbol": "cafe"
+    },
+    "geometry": {
+      "type": "Point",
+      "coordinates": [
+        114.420,
+        30.4595
+      ]
+    }
+  }, {
```
Comment (xhacker) [inline wuhan.geojson]: Same here.

Comment (lvwzhen) [inline wuhan.geojson]: oh,sorry. I used the Sublime-HTMLPrettify plugin,let me modify it.

**Comment by xhacker** [comment]
多谢！

**Pull request merged**